styles.css

```diff
@@ -450,3 +461,427 @@ h3 {
 .footer-btn:active {
   background-color: #2230d2;
 }
+
+.header-wrapper {
+  display: none;
+}
+
+.ul-nav-menu {
+  display: none;
+}
+
+.desktop-img {
+  display: none;
+}
+
+.second-section-desktop-img {
+  display: none;
+}
+
+#third-section-desktop-img {
+  display: none;
+}
+
+#subsection-four-desktop-img {
+  display: none;
+}
+
+/*******          MEDIA QUERY      *******/
+@media screen and (min-width: 768px) {
+  .header-container {
+    background-image: url("images-desk/header-background-shape-desktop.svg");
+    width: 1440px;
+    height: 916px;
+    margin: 0 auto;
+  }
+
+  .nav-menu {
+    display: flex;
+    justify-content: space-between;
+    margin-top: 25px;
+  }
+
+  .logo {
+    margin-top: 25px;
+    margin-left: 155px;
+  }
+
+  .menu-icon {
+    display: none;
+  }
+
+  .ul-nav-menu {
+    display: flex;
+    list-style: none;
+    margin-top: 25px;
+  }
+
+  .ul-nav-menu li {
+    margin-left: 16px;
+    margin-right: 16px;
+  }
+
+  .main-heading h1 {
+    font-size: 48px;
+    font-style: normal;
+    font-weight: 700;
+    height: 122px;
+    width: 920px;
+    display: flex;
+    align-items: center;
+    margin-bottom: 12px;
+    margin-left: 260px;
+    margin-right: 260px;
+    margin-top: 314px;
+    color: #172b4d;
+  }
+
+  p {
+    width: 920px;
+    height: 98px;
+    margin-top: 12px;
+    margin-left: 260px;
+    margin-right: 260px;
+    font-size: 20px;
+    font-weight: 400;
+    text-align: left;
+    color: #344563;
+  }
+
+  h3 {
+    margin-top: 12px;
+    margin-bottom: 12px;
+    margin-left: 260px;
+    color: #7f8cff;
+    font-weight: 500;
+  }
+
+  .social-icons ul {
+    display: flex;
+    list-style: none;
+    justify-content: flex-start;
+    flex-direction: row;
+    margin-left: 260px;
+  }
+
+  .works-section {
+    display: grid;
+    row-gap: 142px;
+    width: 1440px;
+    height: 2694px;
+    padding-left: 142px;
+    padding-right: 142px;
+    padding-top: 142px;
+    padding-bottom: 142px;
+    margin: 0 auto;
+    background-color: #e5e5e5;
+  }
```

Comment:
- [ ] Kindly be noticed that there is a horizontal scroll in the desktop size of your portfolio so suggested changing the width in some divs from 1440px to 100% also remove some padding that causes this horizontal scroll.

![Screenshot from 2022-04-29 20-33-15](https://user-images.githubusercontent.com/33752735/166007029-ed2bbe4f-1941-4869-8f44-c52d54eb63bf.png)